src/style.css

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(./images/star.svg) no-repeat center;
+}
+
+.stars__star.is-active {
```

Comment:
don't use extra class for stars, use `nth-child` instead

Reply:
Extra classes for start were deleted.